src/main.rs

```diff
@@ -128,6 +133,10 @@ ERROR message:
                 .await
                 .context(format!("toggle-mute {input}"))?;
         }
+        Command::GetMute { input } => {
+            let muted = client.inputs().muted(InputId::Name(&input)).await?;
```

Comment:
Let's add a `.context()` like in the other match arms

Reply:
Exactly the same as we do for toggle-mute for example (you may need to pull):
https://github.com/jonhoo/obs-do/blob/131f707a15f8a435128ee2f6c07a44e7e6ae8832/src/main.rs#L125-L129